### DISCUSSION TITLE
Release workflow starting automatically

### DISCUSSION THREAD

**Discussion by dynamicat** [community/community#23437] [New to GitHub]
<p>We have two workflows:<br>
test.yml<br>
release.yml</p>
<p>The test.yml workflow is triggered by a push to the develop and master branches like below with also paths defined:</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - master
      - develop
    paths:
      - .github/workflows/*.yml
      - package.json
      - phpinclude/composer.*
      - ruleset.xml
      - "**/*.html"
      - "**/*.php"
      - "**/*.css"
      - "**/*.scss"
      - "**/*.js"
</code></pre>
<p>The release.yml is triggered with the creation of a tag like so:</p>
<pre><code class="lang-auto">name: release
on:
  create:
    tags:
      - '*.*.*'

</code></pre>
<p>When a new feature branch is pushed to the repository the test workflow triggers but also for some reason the release workflow starts on its own, even though we have specified for the workflow to be triggered manually upon tag creation.</p>
<p>What could make this happen and/or is this normal?</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>Try using the Release keyword not the Create keyword:</p>
<p><a href="https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release</a></p>
<p>Create triggers when a branch or a tag is created. When you create that new branch, it triggers the release workflow as well.</p>

**Comment by dynamicat** [comment]
<p>I tried this also and nothing.  With this workflow we dont want it to run automatically at all.  It is triggered only when a tag is created.</p>
<p>What would be your suggestion? <a class="mention" href="/mickeygousset">@mickeygousset</a></p>

**Comment by mickeygousset** [comment]
<p>Hey I’ll try and look at this later tonight when I’m done with my “day job”. :slight_smile:</p>
<p>Can you send me a slightly more detailed idea of your process? For example, are you creating your tags in Git at the command line and pushing them?  Or are you creating them in the GitHub web interface?</p>
